README.md

```diff
@@ -1,84 +1,55 @@
-$api->payment->all($options); // Returns array of payment objects
-$payment = $api->payment->fetch($id); // Returns a particular payment
-$api->payment->fetch($id)->capture(array('amount'=>$amount)); // Captures a payment
-$api->refund->create(array('payment_id' => $id)); // Creates refund for a payment
-$api->refund->create(array('payment_id' => $id, 'amount'=>$refundAmount)); // Creates partial refund for a payment
-$refund = $api->refund->fetch($refundId); // Returns a particular refund
-$card = $api->card->fetch($cardId); // Returns a particular card
-$customer = $api->customer->create(array('name' => 'Razorpay User', 'email' => 'customer@razorpay.com')); // Creates customer
-$customer = $api->customer->fetch($customerId); // Returns a particular customer
-$api->customer->edit(array('name' => 'Razorpay User', 'email' => 'customer@razorpay.com')); // Edits customer
-$token = $api->customer->token()->fetch($tokenId); // Returns a particular token
-$api->customer->token()->all($options); // Returns array of token objects
-$api->customer->token()->delete($tokenId); // Deletes a token
-
-// To get the payment details
-echo $payment->amount;
-echo $payment->currency;
-// And so on for other attributes
 ```
+composer require razorpay/razorpay
+```
+
+- **Manually**
 
-For further help, see our documentation on <https://docs.razorpay.com>.
+Download the [latest release](https://github.com/razorpay/razorpay-php/releases) and then include `Razorpay.php` in your application, after which you can use the API as usual.
 
-[composer-install]: https://getcomposer.org/doc/00-intro.md#installation-linux-unix-osx
+## Docs
 
-## Developing
+You'll find the entire & thorough documentation for this package in the [WIKI](https://github.com/razorpay/razorpay-php/wiki). The complete API docs can be found on our [website](https://docs.razorpay.com/docs/getting-started).
 
-See the [doc.md](doc.md) file for getting started with development.
+Stuck somewhere using the Razorpay PHP package, any feature requests, or an unsolicited advice? Feel free to [create an issue](https://github.com/razorpay/razorpay-php/issues), we will try to address it as soon as possible.
 
 ## License
 
-The Razorpay PHP SDK is released under the MIT License.
+The Razorpay PHP package is released under the [MIT License](LICENSE.txt).
+
+## Contributing
+
+Thank you for considering contributing to the Razorpay PHP package. You can read the contribution guide lines [here](contributing.md)
 
-## Release
+## Security
 
-Steps to follow for a release:
+If you discover any security related issues, please email to [support@razorpay.com](mailto:support@razorpay.com) instead of using the issue tracker.
+
+## Credits
+
+- [Contributors](https://github.com/razorpay/razorpay-php/contributors)
+
+## License
+Razorpay PHP package is open-sourced software licensed under the [MIT license](LICENSE.txt)
 
-0. Merge the branch with the new code to master.
```

Comment:
While @razorpay is owner for the package/repo, we are still open to contributions from everyone. Keeping the release instructions on open-source projects is what has helped us standardize the process. This is also something other projects follow.

What would you suggest as an alternative?